_posts/2017-04-04-things-the-rust-community-team-does.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Things the Rust Community Team does"
+author: Florian Gilcher
+description: "Describing the Rust Community Team work"
+---
+
+A common question that we get is "what precisely does the Rust community team do?" and probably the member you are currently speaking to has no straight answer to you. That has a reason: the team does a lot of things and any individual often just does a couple of them. Many of the projects that come out of this are then taken on by individual teams and are not necessarily visible as community team action. An important part is to discuss our and others ideas and support each other in our endeavours.
```

Comment:
I'm going to give you a different answer :) The way I think about it is:

The community team is all about bringing together and supporting the Rust organizers from around the world. We all work together to to build global movements to make our language more approachable, and our community more welcoming and fun. All our welcome to participate with our initiatives, and we would love to help mentor and support anyone in our community to create their own.